workspaces/global-header/.changeset/lazy-singers-fly.md

```diff
@@ -0,0 +1,5 @@
+---
+'@red-hat-developer-hub/backstage-plugin-global-header-test': patch
+---
+
+Initial version of this global-header test-plugin.
```

Comment:
Isn't it be `global-header-test plugin` instead 😅

Reply:
simplified